airbyte-integrations/connectors/source-facebook-pages/tools/schema_gen.py

```diff
@@ -0,0 +1,484 @@
+    ]                                                                           "properties": {
+}                                                                                   "previous": {
+                                                                                        "type": ["string", "null"]
+                                                                                    },
+                                                                                    "next": {
+                                                                                        "type": ["string", "null"]
+                                                                                    },
+                                                                                    "cursors": {
+                                                                                        "type": "object",
+                                                                                        "properties": {
+                                                                                            "before": {
+                                                                                                "type": ["string", "null"]
+                                                                                            },
+                                                                                            "after": {
+                                                                                                "type": ["string", "null"]
+                                                                                            }
+                                                                                        }
+                                                                                    }
+                                                                                }
+                                                                            }
+                                                                        }
+                                                                    }
+                                                                }
+
+"""
+
+import json
+import os
+
+spec_path = "facebook-business-sdk-codegen/api_specs/specs"
+fb_node_files = os.listdir(spec_path)
+fb_node_files.sort()
+FB_NODES = [name.split(".")[0] for name in fb_node_files]
+
+print(f"FB SDK identifies specs for {len(FB_NODES)} nodes in ./{spec_path}")
+
+FB_TYPE_DEFAULT = {"type": ["string", "null"]}
+
+FB_TYPES = {
+    "int": {"type": ["integer", "null"]},
+    "int32": {"type": ["integer", "null"]},
+    "unsigned int32": {"type": ["integer", "null"]},
+    "unsigned int": {"type": ["integer", "null"]},
+    "float": {"type": ["integer", "null"]},
+    "string": {"type": ["string", "null"]},
+    "numeric string": {"type": ["string", "null"]},
+    "datetime": {"type": ["string", "null"]},
```

Comment:
would it be helpful to include the date-time format hint?

Reply:
you are right, it is updated now